components/account/AccountButton.tsx

```diff
@@ -246,14 +248,14 @@ const AccountButton: FC<{
                           <BalanceRow
                             imgPath="/currencies/ztg.jpg"
                             units={constants?.tokenSymbol}
-                            balance={activeBalance}
+                            balance={activeBalance ?? undefined}
```

Comment:
think it's better to make `activeBalance` be `Decimal | undefined` by modifying `useBalance` hook
e.g. 

```
  const query = useQuery<Decimal | undefined>(
  [...],
async() => {
....
/// no return at the end
}, {}):
```